apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -110,21 +109,49 @@ const AttestationsTable = ({
       {
         header: 'Status',
         accessorFn: contribution =>
-          contribution.attestations.filter(attestation => {
-            return attestation.user.id === userData?.id;
-          })[0]?.attestation_status?.name || 'Unattested',
-        cell: ({ getValue }: { getValue: Getter<string> }) => {
+          contribution.guilds[0]?.verified ? 'Verified' : 'Unverified',
+        cell: ({
+          getValue,
+          row,
+        }: {
+          getValue: Getter<string>;
+          row: Row<UIContribution>;
+        }) => {
           const status = getValue();
-          return (
-            <Text textTransform="capitalize">
-              {status}{' '}
-              <span
-                role="img"
-                aria-labelledby="Emoji indicating contribution status: Sun emoji for minted and Eyes emoji for staging."
-              >
-                {statusEmojiSelect(status)}
-              </span>{' '}
-            </Text>
+          const statusMapHover =
+            status === null
+              ? 'noFramework'
+              : status === 'Verified'
+              ? 'Verified'
+              : 'Unverified';
```

Comment:
Chaining turnarys is kind of tough to read. An if/else statement may be more readable

Reply:
Good call, will work on that